src/activities/quizzes/attempts/QuizAttemptsEntity.js

```diff
@@ -92,6 +102,18 @@ export class QuizAttemptsEntity extends Entity {
 		return entity.hasActionByName(Actions.quizzes.attempts.updateRetakeIncorrectOnly);
 	}
 
+	/**
+	 * @returns {bool} can update quiz attempt conditions
+	 */
+
+	canUpdateAttemptConditions() {
+		const entity = this.getAttemptConditionsSubEntity();
+		if (!entity) return;
+		const subentity = entity.getSubEntityByClass(Classes.quizzes.attempts.attemptCondition); // gets first matching sub-entity

```

Comment:
is there always at least 1 attempt condition? If there isn't, how will we determine if we can edit attempt conditions?

Reply:
> is there always at least 1 attempt condition?
No
> If there isn't, how will we determine if we can edit attempt conditions?
Will follow up on this